docs/website/content/2.configuration/4.block/backend_tls.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'TLS (Backend)'
+description: 'TLS settings for the related backend.'
+draft: false
+---
+
+# TLS (Backend)
+
+| Block name   | Context                                       | Label    |
+|:-------------|:----------------------------------------------|:---------|
+| `tls`        | [Backend Block](/configuration/block/backend) | no       |
+
+Couper has a command-line argument to add a `ca-file` to the backend CA-Pool for all backends.
+However, this `tls` block allows a more specific pool configuration per backend if the `server_ca_certificate` or
+`server_ca_certificate_file` is provided.
+
+### mTLS
+
+Additionally the `client_certificate`(or `client_certificate_file`) and `client_private_key` (or `client_private_key_file`)
+attributes allow the backend to present those ones during a TLS handshake to an origin which requires them due to an mTLS setup.
```

Comment:
"allow the backend to present those ones during"
-> "allow the backend to present certificate and key during"